clin_msi/msi_model_scripts/apply_msi_model.py

```diff
@@ -175,3 +176,29 @@ def apply_model(infile,moddir,outfile,normalization_scheme,shap_plot_dir=None):
             shaprec=testdat.iloc[i]
             shapdict_loc=bmdict[i]
             build_n_save_shap_plot(shaprec,shapdict_loc,shap_outfile)
+
+def apply_model_trunc(infile,moddir,outfile,normalization_scheme,shap_plot_dir=None):
+    df=pd.read_csv(infile)
+    sampcol='SAMPLE_NAME'
+    if sampcol not in df:
+        df[sampcol]=['SAMPLE_' + str(i+1) for i in range(len(df))]
+    df=df.rename(columns={sampcol:'samp'})
+    moddir='/igm/home/jbg001/git/msi/dat/mods_std_091321'
```

Comment:
Can you get rid of this hard coded path? the moddir will be passed to clin-msi at runtime.